pkg/webui/oauth.js

```diff
@@ -25,6 +26,12 @@ import Init from './lib/components/init'
 
 import createStore from './oauth/store'
 
+WebFont.load({
```

Comment:
While this fixes the issue of FUOT, it is not what we want, since the webfontloader will load the font from Google. Since we need to support offline mode, we need to continue fetching the font as a local asset.

Reply:
After researching about this for a little more, I found this:
https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display

If that works then it should be the way to go.